test/RepositoryTest.php

```diff
@@ -65,6 +65,29 @@ public function testLoadVersions()
                     'type' => 'drupal-module'
                 ),
             ),
+            'drupal' => array(
+                '7.43.0' => [
+                    'type' => 'drupal-core'
+                ],
+                'dev-8.0.x' => [
+                    'type' => 'drupal-project'
+                ],
+                'dev-8.1.x' => [
+                    'type' => 'drupal-project'
+                ],
+                'dev-8.2.x' => [
+                    'type' => 'drupal-project'
+                ],
+                '8.0.5' => [
+                    'type' => 'druapl-project'
```

Comment:
Is this an accidental or intentional typo? It occurs two more times after this one.